src/test/java/io/stargate/sgv2/jsonapi/service/cqldriver/CqlSessionCacheTest.java

```diff
@@ -0,0 +1,56 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver;
+
+import static io.stargate.sgv2.jsonapi.service.cqldriver.TenantAwareCqlSessionBuilderTest.TENANT_ID_PROPERTY_KEY;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.datastax.oss.driver.internal.core.context.DefaultDriverContext;
+import io.quarkus.test.InjectMock;
+import io.quarkus.test.junit.QuarkusTest;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import io.stargate.sgv2.jsonapi.config.OperationsConfig;
+import jakarta.inject.Inject;
+import java.lang.reflect.Field;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+public class CqlSessionCacheTest {
+
+  private static final String TENANT_ID_FOR_TEST = "test_tenant";
+
+  @InjectMock protected StargateRequestInfo stargateRequestInfo;
+
+  @Inject CQLSessionCache cqlSessionCache;
+
+  @Inject OperationsConfig operationsConfig;
+
+  // TODO @Kathir, what is the difference of this class and FixedTokenTests
+  @Test
+  public void testOSSCxCQLSessionCache() throws NoSuchFieldException, IllegalAccessException {
+
+    //    @Kathir
```

Comment:
Can remove commented out code?

Reply:
@kathirsvn

Reply:
Fixed in this commit 0f5b017a472c17a9eea56aea6b96abf56e8bdada